x-pack/plugins/apm/public/components/shared/time_comparison/index.tsx

```diff
@@ -59,110 +59,126 @@ function formatDate({
   return `${momentStart.format(dateFormat)} - ${momentEnd.format(dateFormat)}`;
 }
 
-function getSelectOptions({
+export function getComparisonTypes({
   start,
   end,
-  rangeTo,
-  comparisonEnabled,
 }: {
   start?: string;
   end?: string;
-  rangeTo?: string;
-  comparisonEnabled?: boolean;
 }) {
   const momentStart = moment(start);
   const momentEnd = moment(end);
 
-  const dayBeforeOption = {
-    value: TimeRangeComparisonType.DayBefore,
-    text: i18n.translate('xpack.apm.timeComparison.select.dayBefore', {
-      defaultMessage: 'Day before',
-    }),
-  };
-
-  const weekBeforeOption = {
-    value: TimeRangeComparisonType.WeekBefore,
-    text: i18n.translate('xpack.apm.timeComparison.select.weekBefore', {
-      defaultMessage: 'Week before',
-    }),
-  };
-
   const dateDiff = Number(
```

Comment:
No need to convert to number (it already is afaict)

```suggestion
  const dateDiff = (
```